johnzon-core/src/main/java/org/apache/johnzon/core/JsonProviderImpl.java

```diff
@@ -53,6 +53,12 @@
 import javax.json.stream.JsonParserFactory;
 
 public class JsonProviderImpl extends JsonProvider implements Serializable {
+
+    /**
+     * This makes Johnzon to be compliant with spec, essentially regarding the JSON Pointer /-
```

Comment:
Given that the TCK is open, I'd suggest referencing the specific test case here. Not so much for a javadoc point of view, but more to track the history. The JSR (as far as I can see) doesn't have any spec text at all, so a developer going looking for it, will at best arrive at the RFC.

The setting is necessary for Johnzon to be compliant with JSR-374 / Jakarta JSON Processing (https://github.com/eclipse-ee4j/jsonp/), which specifies the "/-" reference points to the index _after_ the last element in the array: https://github.com/eclipse-ee4j/jakartaee-tck/tree/master/src/com/sun/ts/tests/jsonp...<whichever test>